source/indexes.txt

```diff
@@ -12,6 +12,32 @@ Manage Indexes
    :depth: 1
    :class: singlecol
 
+Indexes are special data structures that improve query performance.
+Indexes store a portion of a collection's data in an easy-to-traverse 
+form. The index stores the value of a specific field or set of fields, 
+ordered by the value of the field.
+
+To improve query performance, build indexes on fields that appear often 
+in queries and for all operations that 
+:ref:`sort by a field <query-bar-sort>`.
+
+- Queries on an indexed field use the index to limit the number of 
+  documents that must be scanned to find matching documents.
+
+- Sort operations on an indexed field return documents pre-sorted 
```

Comment:
So I think keeping the "can" is important because there are instances where the query might not be able to use the index for some unforeseen circumstance, so we don't want to claim it will always use it. The language used in the server docs is generally "x can use"

- Queries on an indexed field can use the index...

- Sort operations on an indexed field can return documents pre-sorted...